pkg/retry/retry.go

```diff
@@ -16,19 +16,14 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// RetryOptions defines the option to retry
-// revive does not like the name because the package is already called retry
-//nolint:revive
-type RetryOptions struct {
-	MaxRetry int           // The number of times to possibly retry
-	Delay    time.Duration // The delay to use between retries, if set
+// Options defines the option to retry.
+type Options struct {
+	MaxRetry int           // The number of times to possibly retry.
+	Delay    time.Duration // The delay to use between retries, if set.
 }
```

Comment:
Add a `type RetryOptions = Options` to avoid breaking consumers of this package.